eolearn/core/utils/parallelize.py

```diff
@@ -74,10 +72,7 @@ def parallelize(
     :return: A list of function results.
     """
     if not params:
-        raise ValueError(
-            "At least 1 list of parameters should be given. Otherwise it is not clear how many times the"
-            "function has to be executed."
-        )
+        return []
```

Comment:
Just to be clear, you removed the warning and just return the empty list, meaning that nothing is done in the end? Or is the error raised at some later stage?

I guess the warning really isn't necessary, if you don't provide params, there's nothing to be done, period.

Reply:
yeah, i dont know why there was an error there in the first place. It is clear what the result should be